source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 import { observable, action, computed, runInAction } from 'mobx';
 import moment from 'moment';
-import { isEqual, includes, get } from 'lodash';
+// @DECENTRALIZATION TODO: Remove `set`
```

Comment:
@daniloprates Can you remove this comment and apply the improvements mentioned in TODO?